packages/flutter_localizations/lib/src/l10n/material_en.arb

```diff
@@ -151,6 +151,11 @@
     "description": "The label for select-all buttons and menu items."
   },
 
+  "searchTextFieldPlaceholerLabel": "Search",
```

Comment:
I think we should add this to the cupertino_en.arb file. Then the script would add them to the generated_cupertino_localizations.dart file

Reply:
oh noooo I added it to the wrong file